src/component/segments/SegmentList/SegmentList.tsx

```diff
@@ -0,0 +1,144 @@
+                    //@ts-expect-error
+                    setCurrentSegment={setCurrentSegment}
+                    setDelDialog={setDelDialog}
+                />
+            );
+        });
+    };
+
+    const renderNoSegments = () => {
+        return (
+            <div className={styles.container}>
+                <Typography className={styles.title}>
+                    There are no segments created yet.
+                </Typography>
+                <p className={styles.subtitle}>
+                    Segment makes it easy for you to define who should be
+                    exposed to your feature. The segment is often a collection
+                    of constraints and can be reused.
+                </p>
+                <PermissionButton
+                    onClick={() => {}}
+                    variant="outlined"
+                    color="secondary"
+                    className={styles.paramButton}
+                    permission={ADMIN}
+                >
+                    Create your first segment
+                </PermissionButton>
+            </div>
+        );
+    };
+
+    return (
+        <div>
+            <Table>
+                <TableHead>
+                    <TableRow className={styles.tableRow}>
+                        <TableCell classes={{ root: styles.cell }}>
+                            Name
+                        </TableCell>
+                        <TableCell classes={{ root: styles.cell }}>
+                            Description
+                        </TableCell>
+                        <TableCell classes={{ root: styles.cell }}>
+                            Created on
+                        </TableCell>
+                        <TableCell classes={{ root: styles.cell }}>
+                            Created By
+                        </TableCell>
+                        <TableCell
+                            align="right"
+                            classes={{ root: styles.cell }}
+                        >
+                            {hasAccess(ADMIN) ? 'Actions' : ''}
+                        </TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>{segments.length > 0 && renderSegments()}</TableBody>
+
+                <PaginateUI
+                    pages={pages}
+                    pageIndex={pageIndex}
+                    setPageIndex={setPageIndex}
+                    nextPage={nextPage}
+                    prevPage={prevPage}
+                />
+            </Table>
+            {segments.length === 0 && renderNoSegments()}
+            <SegmentDeleteConfirm
+                //@ts-expect-error
+                segment={currentSegment}
+                open={delDialog}
+                setDeldialogue={setDelDialog}
+                handleDeleteSegment={onDeleteSegment}
+                confirmName={confirmName}
+                setConfirmName={setConfirmName}
+            />
+            <br />
```

Comment:
Instead of using a `<br />` for styling, how about some extra padding on the container?